content/english/anti-vegan/humans-are-superior.md

```diff
@@ -0,0 +1,84 @@
+>   
+> Yes, humans are unique and distinct from other animals in our capacity for complex problem solving. However, pick any species and you will likely find an attribute that also makes them special.  
+>   
+> No unique set of abilities or characteristics, in itself, provides a criterion for ‘superiority’. 
+For humans to claim that humans are superior because we have specific unique attributes is just another form of prejudice.  
+>  
+> Even if *it is the case* we are uniquely superior, morality requires we care for those less capable than ourselves; not that we treat them as a resource.
+
+## Detail - Analytic
+
+> A Definition of Superior (they are many and varied) : of higher rank, quality, or importance - [Merriam-Webster](http://web.archive.org/web/20190711165845/https://www.merriam-webster.com/dictionary/superior)
+
+
+What most people forget (*or choose to overlook*) in using the term superior is that:  
+
+
+1. we determine the ranking systems, 
+2. we determine what denotes better quality, and 
+3. we determine what makes one thing more important than another.
+
+> On the planet Earth, man had always assumed that he was more intelligent than dolphins because he had achieved so much—the wheel, New York, wars and so on—whilst all the dolphins had ever done was muck about in the water having a good time. But conversely, the dolphins had always believed that they were far more intelligent than man—for precisely the same reasons.  
+> - Douglas Adams *from* Hitchhiker's Guide to the Galaxy, 1979
+
+### Humans are different to other animals
+
+That’s true. Humans have a capacity for complex problem solving that is not present in other animals. But why should a set of unique abilities or characteristics equate to superiority and give us the right to do what we like to those deemed inferior?
+
+> Until recently it was considered that our capacity for abstract thought made us unique. However, “there is growing literature indicating that animals are capable of causal reasoning and abstraction”  
+> [Decision-Making From the Animal Perspective: Bridging Ecology and Subjective Cognition (2019)](https://www.frontiersin.org/articles/10.3389/fevo.2019.00164/full)  
+
+Difference and uniqueness are not synonymous with superiority. 
+
+### The flawed reasoning of Superiority and Domination
+
+> For any X and Y, if X is superior (or morally superior) to Y, then X is morally justified in subordinating Y.
+
+This logic is used to claim that humans are superior to, and may dominate, non-humans *due to some difference* that is elevated (by humans) in importance. However, without the "assumptions  that *humans are superior* to (at least some) nonhumans, and that *superiority justifies subordination*, all one has is some difference" - Karen J. Warren *from* The power and the Promise of Ecological Feminism *in* Environmental Ethics.
```

Comment:
Non-humans and nonhumans. Was it quoted this way? Also written nonhumans in a later quote.

Reply:
Yeah, quote verbatim. Yeah it's used both ways.  I could lose the hyphenated one if need be? Won't make much odds to most readers

Reply:
I just like to see things tidy but as you say, it really doesn't matter so I leave it up to you :)

Reply:
Think there was only a single occurrence of non-  so changed it